package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
+    "prestart": "npm run build",
```

Comment:
Owh, I added that myself, I just forgot to remove it again. It's just for my convinience so it will always  build before starting the bot. But I'll remove it.

Reply:
Just to check, during development you can run `npm run dev`, which will run and live-reload the app. :)